main.nf

```diff
@@ -11,7 +11,7 @@ tertiarySoftwareReport = 'None'
 
 if ( params.containsKey('tertiaryWorkflow' )){
     tertiaryWorkflow = params.tertiaryWorkflow
-    if ( tertiaryWorkflow == 'scanpy-galaxy' ){
+    if ( tertiaryWorkflow == 'scanpy-workflow' ){
```

Comment:
This is breaking other part of code as `scanpy-workflow` was already an option and it expects our new wf to be in line with others which is not at the moment 

we could replace it with `scanpy-tertiary`

Reply:
for example its breaking ` process make_tertiary_software_report` will break other steps as well